LinksAwakeningClient.py

```diff
@@ -741,8 +742,8 @@ async def deathlink():
                 await asyncio.sleep(1.0)
 
 def run_game(romfile: str) -> None:
-    auto_start = typing.cast(typing.Union[bool, str],
-                            Utils.get_options()["ladx_options"].get("rom_start", True))
+    auto_start = typing.cast(typing.Union[bool, str], LinksAwakeningWorld.settings.rom_start)

```

Comment:
the cast is also unnecessary since the settings api will already give it proper typing of `RomStart | bool` (where RomStart is a custom str subclass)
```suggestion
    auto_start = LinksAwakeningWorld.settings.rom_start
```